### DISCUSSION TITLE
Is this package works with AzureOpenAI?

### DISCUSSION THREAD

**Discussion by mraguth** [microsoft/graphrag#327] [Q&A] [answered]
Please share llm and text_embedder configurations along with import settings please? 

In our corporation, we use Azure OpenAI and love to explore.

**Comment by mraguth** [comment]
I am able to configure and working for me now :-)

**Reply by jgbradley1** [reply, marked as answer]
Azure OpenAI is supported. With Azure OpenAI specifically, there are two authentication options. You can provide an API key or if you leave the API key out of your configuration file, `graphrag` will fallback and attempt to use managed identity if you have it setup on your system.

**Reply by ghost** [reply]
Hi @jgbradley1 ,

Can you help me to configure by giving sample file..
what is organization parameter. is it api key for the organization?

encoding_model: cl100k_base
skip_workflows: []
llm:
  ----# api_key: ${GRAPHRAG_API_KEY}
  type: azure_openai_chat
  model: gpt-35-turbo-16k
  model_supports_json: false # recommended if this is available for your model.
  ----# max_tokens: 4000
  ---- # request_timeout: 180.0
  api_base: https://xxxxx.openai.azure.com/
  api_version: 2024-02-15-preview
  organization: 893xxxxxc675e2
  deployment_name: gpt-35-turbo-16k

-------------Received ERROR ---------------
File \"C:\\Users\\vijay.r\\anaconda\\envs\\ragagent\\lib\\site-packages\\openai\\_base_client.py\", line 1514, in request\n    return await self._request(\n  File \"C:\\Users\\vijay.r\\anaconda\\envs\\ragagent\\lib\\site-packages\\openai\\_base_client.py\", line 1610, in _request\n    raise self._make_status_error_from_response(err.response) from None\nopenai.AuthenticationError: Error code: 401 - {'statusCode': 401, 'message': 'Unauthorized. Access token is missing, invalid, audience is incorrect (https://cognitiveservices.azure.com), or have expired.
